comments/models.js

```diff
@@ -13,8 +13,23 @@ exports.selectCommentsByArticleId = (article_id) => {
             if (result.rows.length === 0) {
                 return []
             }
-            console.log(result.rows)
             return result.rows
         })
     })
-} 
\ No newline at end of file
+} 
+
+exports.insertComment = (username, body, article_id) => {
+    const queryStr = `
+    INSERT INTO comments (body, article_id, author) 
+    VALUES ($1, $2, $3) 
+    RETURNING *
+    ;`
+
+    return checkArticleExists(article_id).then(() => {
```

Comment:
Do you need to check the article exists? You may find the foreign key will handle that issue when you try to insert to an article that doesn't exist